packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/iotSiteWiseQueryEditor.spec.tsx

```diff
@@ -0,0 +1,439 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IoTSiteWiseQueryEditor } from './iotSiteWiseQueryEditor';
+import { resourceExplorerQueryClient } from '@iot-app-kit/react-components';
+import {
+  AssetModelPropertySummary,
+  AssetPropertySummary,
+  AssetSummary,
+  IoTSiteWise,
+  ListAssetsResponse,
+  ListAssetPropertiesResponse,
+  ListAssetModelPropertiesResponse,
+} from '@aws-sdk/client-iotsitewise';
+import { configureDashboardStore } from '~/store';
+import { Provider } from 'react-redux';
+
+export async function waitForLoadingToFinish() {
+  await waitFor(() => {
+    expect(screen.getByText(/Loading/)).toBeVisible();
+  });
+
+  await waitFor(() => {
+    expect(screen.queryByText(/Loading/)).not.toBeInTheDocument();
+  });
+}
```

Comment:
Instead of duplicating, import this and the other utilities from react-components or move them to a common place.